lang/java/reef-wake/wake/src/test/java/org/apache/reef/wake/test/avro/ProtocolSerializerTest.java

```diff
@@ -126,14 +126,13 @@ private RemoteManager getTestRemoteManager(final String identifier) {
         localAddressProvider, tcpPortProvider);
```

Comment:
just do `return remoteManagerFactory.getInstance(...)` here and then we don't need that `remoteManager` variable at all